INFO - 2024-03-31 13:34:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-31 13:34:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-31 13:34:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-31 13:35:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-31 13:36:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-31 13:36:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-31 13:37:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-03-31 13:37:09 --> Error: Class "null" not found
in VENDORPATH\myth\auth\src\Config\Services.php on line 81.
 1 SYSTEMPATH\Config\BaseService.php(258): Myth\Auth\Config\Services::activator(Object(Myth\Auth\Config\Auth), false)
 2 SYSTEMPATH\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('activator', [...])
 3 VENDORPATH\myth\auth\src\Config\Services.php(74): CodeIgniter\Config\BaseService::getSharedInstance('activator', null)
 4 SYSTEMPATH\Config\BaseService.php(258): Myth\Auth\Config\Services::activator()
 5 SYSTEMPATH\Common.php(1008): CodeIgniter\Config\BaseService::__callStatic('activator', [])
 6 VENDORPATH\myth\auth\src\Controllers\AuthController.php(184): service('activator')
 7 SYSTEMPATH\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\codenighter\\public\\index.php')
INFO - 2024-03-31 15:11:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-31 15:11:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-31 15:11:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-31 15:12:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-03-31 15:12:07 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\codenighter\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\codenighter\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'crud_data', 3306, '', 0)
#1 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codenighter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codenighter\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('Admin', 'username', Array, NULL, 'username')
#6 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'Admin', Array, Array)
#7 C:\xampp\htdocs\codenighter\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\codenighter\vendor\myth\auth\src\Controllers\AuthController.php(154): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
#10 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#11 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codenighter\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\codenighter\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\codenighter\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codenighter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\codenighter\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('Admin', 'username', Array, NULL, 'username')
#5 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'Admin', Array, Array)
#6 C:\xampp\htdocs\codenighter\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\codenighter\vendor\myth\auth\src\Controllers\AuthController.php(154): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
#9 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#10 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\codenighter\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\codenighter\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-03-31 15:12:07 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('Admin', 'username', [...], null, 'username')
 5 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'Admin', [...], [...])
 6 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 7 VENDORPATH\myth\auth\src\Controllers\AuthController.php(154): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\codenighter\\public\\index.php')
INFO - 2024-03-31 15:16:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-03-31 15:16:13 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\codenighter\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\codenighter\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'crud_data', 3306, '', 0)
#1 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codenighter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codenighter\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('Admin', 'username', Array, NULL, 'username')
#6 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'Admin', Array, Array)
#7 C:\xampp\htdocs\codenighter\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\codenighter\vendor\myth\auth\src\Controllers\AuthController.php(154): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
#10 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#11 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codenighter\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\codenighter\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\codenighter\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codenighter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\codenighter\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('Admin', 'username', Array, NULL, 'username')
#5 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'Admin', Array, Array)
#6 C:\xampp\htdocs\codenighter\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\codenighter\vendor\myth\auth\src\Controllers\AuthController.php(154): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
#9 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#10 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\codenighter\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\codenighter\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-03-31 15:16:13 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `bb_users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('Admin', 'username', [...], null, 'username')
 5 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'Admin', [...], [...])
 6 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 7 VENDORPATH\myth\auth\src\Controllers\AuthController.php(154): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\codenighter\\public\\index.php')
INFO - 2024-03-31 15:17:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-03-31 15:17:04 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\codenighter\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\codenighter\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'crud_data', 3306, '', 0)
#1 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codenighter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codenighter\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('Admin', 'username', Array, NULL, 'username')
#6 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'Admin', Array, Array)
#7 C:\xampp\htdocs\codenighter\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\codenighter\vendor\myth\auth\src\Controllers\AuthController.php(154): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
#10 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#11 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codenighter\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\codenighter\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\codenighter\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codenighter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\codenighter\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('Admin', 'username', Array, NULL, 'username')
#5 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'Admin', Array, Array)
#6 C:\xampp\htdocs\codenighter\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\codenighter\vendor\myth\auth\src\Controllers\AuthController.php(154): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
#9 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#10 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\codenighter\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\codenighter\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-03-31 15:17:04 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('Admin', 'username', [...], null, 'username')
 5 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'Admin', [...], [...])
 6 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 7 VENDORPATH\myth\auth\src\Controllers\AuthController.php(154): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\codenighter\\public\\index.php')
ERROR - 2024-03-31 15:17:27 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\codenighter\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\codenighter\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'crud_data', 3306, '', 0)
#1 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(1412): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(1431): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\codenighter\system\Database\MigrationRunner.php(757): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\codenighter\system\Database\MigrationRunner.php(161): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\codenighter\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\codenighter\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\codenighter\system\CLI\Console.php(46): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\codenighter\spark(102): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\codenighter\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(1412): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(1431): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\codenighter\system\Database\MigrationRunner.php(757): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\codenighter\system\Database\MigrationRunner.php(161): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\codenighter\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\codenighter\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\codenighter\system\CLI\Console.php(46): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\codenighter\spark(102): CodeIgniter\CLI\Console->run()
#10 {main}
INFO - 2024-03-31 15:17:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-03-31 15:17:33 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\codenighter\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\codenighter\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'crud_data', 3306, '', 0)
#1 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codenighter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codenighter\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('Admin', 'username', Array, NULL, 'username')
#6 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'Admin', Array, Array)
#7 C:\xampp\htdocs\codenighter\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\codenighter\vendor\myth\auth\src\Controllers\AuthController.php(154): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
#10 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#11 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codenighter\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\codenighter\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\codenighter\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codenighter\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codenighter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\codenighter\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('Admin', 'username', Array, NULL, 'username')
#5 C:\xampp\htdocs\codenighter\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'Admin', Array, Array)
#6 C:\xampp\htdocs\codenighter\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\codenighter\vendor\myth\auth\src\Controllers\AuthController.php(154): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
#9 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#10 C:\xampp\htdocs\codenighter\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\codenighter\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\codenighter\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-03-31 15:17:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('Admin', 'username', [...], null, 'username')
 5 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'Admin', [...], [...])
 6 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 7 VENDORPATH\myth\auth\src\Controllers\AuthController.php(154): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\codenighter\\public\\index.php')
INFO - 2024-03-31 15:17:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-31 15:17:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-31 15:17:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-31 15:17:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-31 15:18:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-03-31 15:19:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
